index.js

```diff
@@ -153,8 +159,12 @@ cs.get.hwb = function (string) {
 	return null;
 };
 
-cs.to.hex = function (rgb) {
-	return '#' + hexDouble(rgb[0]) + hexDouble(rgb[1]) + hexDouble(rgb[2]);
+cs.to.hex = function () {
+	var rgba = swizzle(arguments);
+
+	return rgba.length < 4 || rgba[3] === 1
+		? '#' + hexDouble(rgba[0]) + hexDouble(rgba[1]) + hexDouble(rgba[2])
+		: '#' + hexDouble(rgba[0]) + hexDouble(rgba[1]) + hexDouble(rgba[2]) + hexDouble(Math.round(rgba[3] * 255));
```

Comment:
Instead, could you build up the RGB portion of the string, and then check for the alpha case and append to the result if need be? That will help [DRY](https://en.wikipedia.org/wiki/Don't_repeat_yourself) things up.


Some pseudo-code:
```
rgb = r + g + b
if rgb.length >= 4 && rgb[3] < 1
    rgb += a
return rgb
```